testgrid/migrations/tables/testinstance.yaml

```diff
@@ -13,6 +13,8 @@ spec:
           type: varchar(255)
         - name: testrun_ref
           type: varchar(255)
+        - name: test_name
+          type: varchar(255)
```

Comment:
I think this needs to either be nullable (in which case you need to handle it in the getter code) or have a default